zsh-aws-vault.plugin.zsh

```diff
@@ -16,44 +16,61 @@ alias avs='aws-vault server'
 # Convenience Functions                                              #
 #--------------------------------------------------------------------#
 function avl() {
-  aws-vault login $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login $aws_profile
 }
 
 function avll() {
-  aws-vault login -s $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login -s $aws_profile
+}
+
+function aven() {
```

Comment:
I like the idea behind this. I've definitely run into it a few times in the past.

`aven` feels related to `ave` when, to the user, they don't really need to know that you're doing an `aws-vault` exec under the hood. What do you think about, instead, calling this `avn` ("aws-vault new")?

Reply:
what this really does is instead of spawning subshell it exports tokens to current shell hence aven - Aws-Vault ENv ;) but i dont really care how its called :)

Reply:
Oh, alright - I see where you were going with this. I like `avenv`!